modules/social_features/social_album/src/Form/SocialAlbumPostForm.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Drupal\social_album\Form;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\node\NodeInterface;
+use Drupal\social_post\Form\PostForm;
+
+/**
+ * Class SocialAlbumPostForm.
+ *
+ * @package Drupal\social_album\Form
+ */
+class SocialAlbumPostForm extends PostForm {
+
+  /**
+   * The visibility options mapping.
+   *
+   * The associative array where keys are node options and values are the
+   * corresponding post options.
+   *
+   * @see field.storage.node.field_content_visibility.yml
+   * @see field.storage.post.field_visibility.yml
+   */
+  const VISIBILITY_MAPPING = [
```

Comment:
Why you can not get this option from the config itself? Because, in this case, it seems, we don't support Social role visibility module.